### PR TITLE
DWR-443 - more unit tests

### PR DESCRIPTION
**Pull request by agorina**
I think this change completes this JIRA.

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/ingest/common/util/ParserHelper.java [hunk 1]
```diff
@@ -126,13 +118,7 @@ public String validate(final String name, final String rawValue, final int maxLe
     public static final Function<String, Integer> toIntegerOrNull = value ->
             StringUtils.hasText(value) ? Integer.parseInt(value.trim()) : null;
 
-    public static final Function<String, LocalDate> tryToParseToLocalDate = value -> {
-        if (isNullOrEmpty(value)) return null;
-        return LocalDate.parse(value);
-    };
+    public static final Function<String, LocalDate> toLocalDateOrNull = value ->
```
Comment (agorina) [inline common/src/main/java/org/opentestsystem/rdw/ingest/common/util/ParserHelper.java]: Renamed to be consistent with other names in this file.

File: common/src/main/java/org/opentestsystem/rdw/ingest/common/util/ParserHelper.java [hunk 2]
```diff
@@ -126,13 +118,7 @@ public String validate(final String name, final String rawValue, final int maxLe
...
+            StringUtils.hasText(value) ? LocalDate.parse(value.trim()) : null;
 
-    public static final Function<String, Boolean> tryToParseToBoolean = value -> {
```
Comment (agorina) [inline common/src/main/java/org/opentestsystem/rdw/ingest/common/util/ParserHelper.java]: This is the same as toBooleanOrNull.

File: common/src/main/java/org/opentestsystem/rdw/ingest/common/util/Precondition.java [hunk 3]
```diff
@@ -15,7 +16,7 @@
      * @throws IllegalArgumentException if the given value is blank
      */
     public static String checkNotBlank(final String value) {
-        checkArgument(!isNullOrEmpty(value), "value may not be blank");
```
Comment (agorina) [inline common/src/main/java/org/opentestsystem/rdw/ingest/common/util/Precondition.java]: This was wrong.

Comment (malaffoon) [inline common/src/main/java/org/opentestsystem/rdw/ingest/common/util/Precondition.java]: Yeah, that surprised me too ... i just assumed isNullOrEmpty worked like the lang3 utility.

**Pull request merged**